### DISCUSSION TITLE
Frame-Page.Component modulo nao encontrado

### DISCUSSION THREAD

**Discussion by ernaneSNcarvalho** [balta-io/7187#50] [Perguntas e Respostas] [answered]
Bom Dia Balta,

Criei a frame-page.component.html manualmente e apos isso so o navbar e renderizado na pagina, aparece o seguinte erro: 
==> Link do projeto no github: https://github.com/ernaneSNcarvalho/petshop

./src/app/app-routing.module.ts:6:0-84 - Error: Module not found: Error: Can't resolve './pages/master/frame-page/frame-page.component' in 'C:\dev\petshop\src\app'

Error: src/app/app-routing.module.ts:7:36 - error TS2307: Cannot find module './pages/master/frame-page/frame-page.component' or 
its corresponding type declarations.

7 import { FramePageComponent } from "./pages/master/frame-page/frame-page.component";

**Comment by andrebaltieri** [comment, marked as answer]
@ernaneSNcarvalho manda um print da sua estrutura de pastas e arquivos... ele não encontrou seu arquivo frame-page.component.ts.
Outro ponto.. após criar manualmente, você parou a aplicação e rodou novamente?

**Reply by ernaneSNcarvalho** [reply]
@andrebaltieri sim parei a aplicacao e tentei outras vezes. Segue print com estrutura das pastas.

![image](https://user-images.githubusercontent.com/66145790/137727383-e1381acc-04ec-45d2-9241-31c711ad1d8c.png)
![image](https://user-images.githubusercontent.com/66145790/137727459-82dcbe1a-8128-4b14-9e65-23b86510bc3d.png)
![image](https://user-images.githubusercontent.com/66145790/137727543-1d3d9d0a-4ffd-463a-9421-d486e356dacb.png)

 
![image](https://user-images.githubusercontent.com/66145790/137727608-a2fa4b45-30bc-44f6-81f2-c53e05894ff0.png)

**Reply by andrebaltieri** [reply]
E como está o código que chama o arquivo que não foi encontrado?
